src/asset_folder_importer/pluto/assetfolder.py

```diff
@@ -33,7 +34,7 @@ def new_connection(self,scheme="http"):
 
     def find_assetfolder(self, path):
         authstring = "{0}:{1}".format(self._user, self._passwd)
-        auth = base64.b64encode(authstring.encode("UTF-8"))
+        auth = always_string(base64.b64encode(authstring.encode("UTF-8")))
```

Comment:
I don't think you need `always_string` here.
`base64.b64encode()` always returns bytes afaik, so you should be able to simply run:

`base64.b64encode(authstring.encode("UTF-8")).decode("UTF-8")`